x-pack/plugins/aiops/common/constants.ts

```diff
@@ -19,3 +19,11 @@ export const RANDOM_SAMPLER_SEED = 3867412;
 export const CASES_ATTACHMENT_CHANGE_POINT_CHART = 'aiopsChangePointChart';
 
 export const EMBEDDABLE_CHANGE_POINT_CHART_TYPE = 'aiopsChangePointChart' as const;
+
+export const LOG_RATE_ANALYSIS_RUN = 'aiops_log_rate_analysis_run';
+
+export const LOG_PATTERN_ANALYSIS_RUN = 'aiops_log_pattern_analysis_run';
+
+export const CHANGE_POINT_DETECTION_RUN = 'aiops_change_point_detection_run';
+
+export const AIOPS_DEFAULT_SOURCE = 'ml_aiops_labs';
```

Comment:
Suggest to make these single consts an enum like object with `as const`, something like:

```
export const AIOPS_TELEMETRY_ID = {
    LOG_RATE_ANALYSIS_RUN: ..., 
    LOG_PATTERN_ANALYSIS_RUN: ...
} as const;
```

Reply:
Updated in bc27b463adabda26003a2e8fc3bbc5b1cf796a62